### PR TITLE
drivers/hih6130: adapt to new i2c API

### PR DESCRIPTION
**Pull request by gebart**
### Contribution description

Adapt drivers/hih6130 to new i2c API

### Issues/PRs references

#6577

### CODE REVIEW

File: drivers/hih6130/hih6130.c [hunk 1]
```diff
@@ -84,7 +84,7 @@ static inline int hih6130_get_humidity_temperature_raw(const hih6130_t *dev, uin
 
     i2c_acquire(dev->i2c);
 
-    if (i2c_read_bytes(dev->i2c, dev->addr, &buf[0], sizeof(buf)) != sizeof(buf)) {
+    if (i2c_read_bytes(dev->i2c, dev->addr, &buf[0], sizeof(buf), 0) != sizeof(buf)) {
```
Comment (smlng) [inline drivers/hih6130/hih6130.c]: should `!= 0`, does not return read bytes anymore

Comment (gebart) [reply drivers/hih6130/hih6130.c]: Oops, will fix

**Comment by dylad** [comment]
@smlng are you happy here ?

File: drivers/hih6130/hih6130.c [hunk 2]
```diff
@@ -84,7 +84,7 @@ static inline int hih6130_get_humidity_temperature_raw(const hih6130_t *dev, uin
 
     i2c_acquire(dev->i2c);
 
-    if (i2c_read_bytes(dev->i2c, dev->addr, &buf[0], sizeof(buf)) != sizeof(buf)) {
+    if (i2c_read_bytes(dev->i2c, dev->addr, &buf[0], sizeof(buf), 0) < 0{
```
Comment (smlng) [inline drivers/hih6130/hih6130.c]: now its missing closing parentheses ☹️

Comment (gebart) [reply drivers/hih6130/hih6130.c]: Ah crap, I edited on my phone through the web interface

**Comment by dylad** [comment]
@smlng can we squash and go ?

**Comment by smlng** [comment]
sure, please squash

**Comment by smlng** [comment]
@gebart ping, please squash

**Comment by aabadie** [comment]
@smlng @gebart, I allowed myself to rebase and squash this PR. Hope this is fine.

**Comment by aabadie** [comment]
All green, let's go!

**Pull request merged**